lib/worker.js

```diff
@@ -94,6 +106,22 @@ async function getLinter(filePath) {
   return tslintDef;
 }
 
+async function getProgram(Linter, configurationPath) {
+  let program;
+  const configurationDir = path.dirname(configurationPath);
+  const tsconfigPath = path.resolve(configurationDir, 'tsconfig.json');
+  try {
+    const stats = await stat(tsconfigPath);
+    const isFile = stats.isFile();
```

Comment:
Since this isn't re-used, and the function name is already clear enough you might as well just move this directly into the conditional.